python/graphstorm/model/node_glem.py

```diff
@@ -161,7 +166,8 @@ def toggle(self, part='lm'):
             raise ValueError(f"Unknown model part: {part}")
 
     def forward(self, blocks, node_feats, edge_feats, labels, input_nodes, use_gnn=True,
-                no_pl=False):
+                no_pl=False, blocks_u=None, node_feats_u=None, edge_feats_u=None,
+                input_nodes_u=None):
```

Comment:
can you explain the new input arguments?

Reply:
will do